source/quick-start.txt

```diff
@@ -37,57 +37,75 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
+.. tabs::
 
-After you verify that you installed the required versions of Node and NPM,
-create a new project.
+   .. tab:: Clone the Project
+      :tabid: clone the project
 
-First, create a directory for your project in your command line interface:
+      First, install git using `Git's Getting Started Guide
+      <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__.
 
-.. code-block:: none
+      Then, clone the `js-starter repository
+      <https://github.com/mongodb-university/js-starter>`__  in your shell:
 
-   mkdir node_quickstart
+      .. code-block:: bash
 
-Then, navigate into that directory so you can work directly with your
-project's files:
+         git clone git@github.com:mongodb-university/js-starter.git 
```

Comment:
If the user doesn't have ssh set up will this work? It might be better to default to https